src/Totals/resources/views/totals.php

```diff
@@ -7,28 +7,26 @@
 ?>
 
 <div class="give-totals">
-	<div class="give-totals__content">
-		<div class="give-totals__message">
-			<?php echo $this->getMessage(); ?>
-		</div>
+	<?php if ( ! empty( $this->getMessage() ) || ! empty( $this->getLinkUrl() ) ) : ?>
+	<p class="give-totals__message">
+		<?php echo $this->getMessage(); ?>
 		<?php if ( ! empty( $this->getLinkUrl() ) && ! empty( $this->getLinkText() ) ) : ?>
-		<div class="give-totals__link">
-			<a href="<?php echo $this->getLinkUrl(); ?>" target="<?php echo $this->getLinkTarget(); ?>"><?php echo $this->getLinkText(); ?></a>
-		</div>
+		<a href="<?php echo $this->getLinkUrl(); ?>" target="<?php echo $this->getLinkTarget(); ?>"><?php echo $this->getLinkText(); ?></a>
 		<?php endif; ?>
-	</div>
+	</p>
+	<?php endif; ?>
 	<?php if ( ! empty( $this->getGoal() ) ) : ?>
 	<div class="give-totals__goal">
 		<div class="give-totals__progress">
 			<?php $percent = ( $this->getTotal() / $this->getGoal() ) * 100; ?>
-			<div class="give-totals__progress-bar" style="width: <?php echo $percent < 100 ? $percent : 100; ?>%"></div>
-			<div class="give-totals__progress-text"> 
-				<?php
-					$total = $this->metric === 'revenue' ? $this->getFormattedTotal() : $this->getTotal();
-					echo sprintf( __( '%1$s of %2$s', 'give' ), $total, $this->getFormattedGoal() );
-				?>
-			</div>
+			<div class="give-totals__progress-bar" style="width: <?php echo $percent < 100 ? $percent : 100; ?>%; background: linear-gradient(180deg, <?php echo $this->getColor(); ?> 0%, <?php echo $this->getColor(); ?> 100%), linear-gradient(180deg, #fff 0%, #ccc 100%);"></div>
```

Comment:
I would consider using [min([ percent, 100 ])](https://www.php.net/manual/en/function.min.php) instead of an inline ternary.

Reply:
Good thinking, done: https://github.com/impress-org/givewp/commit/8003d753c1d9439ba2ae8869d97fcbaf1dfbe281